examples/with-mobx-react-lite/components/StoreProvider.js

```diff
@@ -0,0 +1,39 @@
+import { enableStaticRendering } from 'mobx-react-lite'
```

Comment:
Let's put this file under `lib/store.js` so it looks more similar to the mobx examples and other examples that do something similar.